### ISSUE TITLE
Any suggestions as to how the should the list be organized?

### ISSUE THREAD

**Issue by michaelmcmillan** [michaelmcmillan/awesome-ntnu#6]
Right now it's kind of chaotic. Any suggestions on what categories we should have?

**Comment by relekang** [comment]
Subcategories of websites/webapps is probably a good idea.

**Comment by michaelmcmillan** [comment]
Yep. It would probably be beneficial to differentiate between projects and start-ups as well.
